### PR TITLE
Current and future challenges section

### PR DESCRIPTION
**Pull request by adrn**
Here is a draft outline of the challenges section!

### CODE REVIEW

File: src/ms.tex [hunk 1]
```diff
@@ -1379,24 +1388,19 @@ \subsection{A roadmap for future priorities}
...
+Software communities are facing and addressing similar issues.
+Below we discuss a few of these challenges and ...
+
+\begin{description}
+  \item[Supporting an active group of maintainers] With funding, can pay maintainers. But: time commitment, some positions don't allow. Need for new career paths on the spectrum from software engineer to astronomy researcher. Also need to attract new non-hired contributors to sustain the project, but challenging when the code has become quite complex (astropy as an "institution", don't want to be a walled-off group). (Need for fast feedback and code reviews to empower contributors, but puts a burden on maintainers)
+  \item[Diversity of contributor, maintainer, coordinator communities] Practical issues: translation of content and documentation, github is not accessible in some countries.
+  \item[Long-term / sustained funding for maintaining infrastructure] Further from "science," still critical to astropy, but who will provide updates and maintenance? More than just updating versions and releasing. (grants last 3 years; infrastructure evolves, nature of open source ecosystem)
```
Comment (adrn) [inline src/ms.tex:1401]: @pllim Would you be willing to contribute this text after consulting with the infrastructure team?

File: src/ms.tex [hunk 2]
```diff
@@ -1379,24 +1388,19 @@ \subsection{A roadmap for future priorities}
...
+  \item[Understanding, communicating with, and engaging our user communities] Different overlapping and intersecting communities of users: telescopes, archives, researchers, students, educators, general public. Communities have different needs from Astropy. Also, as ecosystem grows, making sure users know about all of the functionality (workshops, tutorials, educational materials, discoverability, accessibility) (Continued engagement with user community).
```
Comment (adrn) [inline src/ms.tex:1402]: @nmearl would you be willing to write a few sentences to address these issues?

File: src/ms.tex [hunk 3]
```diff
@@ -1379,24 +1388,19 @@ \subsection{A roadmap for future priorities}
...
-\end{itemize}
+Over the last decade, the needs of (and challenges faced by) the \astropy
+Project have evolved, but many common themes persist.
+For example, ...list...
+In many cases, these themes are not unique to \astropy: Many Open-Source
+Software communities are facing and addressing similar issues.
+Below we discuss a few of these challenges and ...
+
+\begin{description}
+  \item[Supporting an active group of maintainers] With funding, can pay maintainers. But: time commitment, some positions don't allow. Need for new career paths on the spectrum from software engineer to astronomy researcher. Also need to attract new non-hired contributors to sustain the project, but challenging when the code has become quite complex (astropy as an "institution", don't want to be a walled-off group). (Need for fast feedback and code reviews to empower contributors, but puts a burden on maintainers)
```
Comment (adrn) [inline src/ms.tex:1399]: @adrn I will do this!

File: src/ms.tex [hunk 4]
```diff
@@ -1340,10 +1340,19 @@ \subsection{Learn Astropy} \label{sec:learn}
 We also plan to look for opportunities to expand the reach of \astropy workshops
 beyond the AAS meetings.
 
+% \begin{itemize}
+% \item {\bf User forums:} The Astropy Project has historically maintained a number of avenues for users to seek help or gain access to the developer community. This includes the Astropy mailing list, the Python in Astronomy Facebook group, and the Astropy Slack workspace. The Moore Foundation grant allows this Slack space to
```
Comment (pllim) [inline src/ms.tex:1344]: Is this like internal notes that you will remove in the final version?

File: src/ms.tex (hunk 1 above)
Comment (pllim) [inline src/ms.tex:1401]: Am I allowed to use line breaks here for `\item`? Hopefully I captured the main points here. Feel free to rephrase.

```suggestion
  \item[Long-term / sustained funding for maintaining infrastructure] Software and packaging infrastructure
  includes but not limited to Continuous Integration (CI), test plugins, documentation builds, and package
  delivery. Keeping the infrastructure working is essential in software development and operations.
  This kind of work has low visibility and is not attractive to scientists or volunteer contributors;
  therefore, it is most effective to delegate such tasks to paid roles. \astropy secured a three-year grant
  for this but the situation remains uncertain once the grant runs out, as infrastructure evolves and
  requires constant maintenance and upgrades. Some of the engineers currently in these roles are
  also employees of research institutions that may leave the Project without much notice, adding to
  the uncertainty factor.
```

File: src/ms.tex (hunk 2 above)
Comment (nmearl) [reply src/ms.tex:1402]: I may need a day or two, but I will get to this!

Comment (adrn) [reply src/ms.tex:1410]: I have some time today, so I'll work on this

**Comment by adrn** [comment]
I have some time today so I'll work on filling in the rest of the text here.

**Comment by adrn** [comment]
@pllim and @nmearl - I filled in some text in the challenges section, and added a new figure. If you have time to read this weekend, would be good to get your eyes on it. But no worries if not - I'm planning to merge this and send out the draft on Monday morning.

**Comment by pllim** [comment]
Whops, sorry I missed this.

**Pull request merged**